frontend/mainwindow.cpp

```diff
@@ -185,6 +185,7 @@ void MainWindow::massSearch(QString name, QString value)
     const char *prompt;
     QString textInput;
     QString text;
+    bool caseSensitivity = false;
```

Comment:
I would argue that storing this in a local variable is unnecessary (or at least a bit convoluted) when you can just probe the ui element for the checked status when you need it. At least that way you're guaranteed consistency.

Reply:
It being part of a dialog means the element is killed upon submission. To access the state post-submission is this not needed? It functions the same as "textInput" above

Reply:
Sorry, you're absolutely right - I missed the context there!